mainnet/X-51-attack/TheLobbyGuy.sol

```diff
@@ -44,7 +98,16 @@ contract TheLobbyGuy {
 
         MiniMeToken votingToken = voting.token();
         uint256 voterStake = votingToken.balanceOfAt(msg.sender, snapshotBlock);
-        uint256 share = yeas / voterStake;
+
+        uint256 balance = _token.balanceOf(address(this));
```

Comment:
This is already provided in `currentBalance` :)

Reply:
True!